projects/cashmere/src/lib/date-range/picker-overlay/picker-overlay.component.html

```diff
@@ -26,19 +25,27 @@
             [invalidDateLabel]="options.invalidDateLabel"
             [mode]="options.mode"
             [hourCycle]="options.hourCycle"
-        >
-        </hc-calendar-wrapper>
+        ></hc-calendar-wrapper>
```

Comment:
I guess Prettier decided to start formatting stuff for me. Sorry for the clutter.